### DISCUSSION TITLE
[Github API V3] Listing repository languages only seems to support Python and C

### DISCUSSION THREAD

**Discussion by Niceplace** [community/community#24550] [Apps, API and Webhooks]
<p>First of all I would like to extend my thanks for the amazing documentation and API that Github provides, it is really useful to us ! </p>
<p>The endpoint to fetch the list of languages for a specific repository only lists “Python” and “C” as the possible responses <a href="https://developer.github.com/v3/repos/#list-languages" rel="nofollow noopener">https://developer.github.com/v3/repos/#list-languages</a></p>
<p>I am wondering if this is by design or maybe it has not been updated ? If we refer to Github’s UI there seems to be a far broader array of languages supported.</p>
<p>If this is indeed an issue and needs to be updated, would it be possible to have a rough estimate of when this could be fixed ?</p>
<p>Cheers !</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/niceplace">@niceplace</a>,</p>
<p>Thank you for being here! That should be an example on our documentation using the <a href="https://developer.github.com/v3/repos/#list-languages" rel="nofollow noopener">https://developer.github.com/v3/repos/#list-languages</a> will list all the languages for the selected repository. For example:</p>
<pre><code>https://api.github.com/repos/andreagriffiths11/spanish-quiz/languages
</code></pre>
<p>Returns:</p>
<pre><code>{
  "Java": 7200
}
</code></pre>
<p>I hope this helps!</p>
